Dockerfile

```diff
@@ -39,8 +39,15 @@ COPY --from=cargo-build /driver /usr/local/bin/driver
 ENTRYPOINT [ "driver" ]
 
 FROM intermediate as orderbook
+RUN apt-get update && \
```

Comment:
Building `heaptrack` is useless since we rebuild it in the final container as well, right?

Reply:
This is required for local tests when I build the orderbook image separately.